src/modules/flow/iio/iio.json

```diff
@@ -421,8 +421,8 @@
            "name": "use_device_default_scale"
          },
          {
-             "data_type": "float",
-             "default": 0,
+             "data_type": "direction-vector",
+             "default": {},
```

Comment:
that is right, but it's not a direction vector. I think it's better to go with separate options, one per channel.